src/views/neutralview/neutral-view-pali.js

```diff
@@ -44,18 +43,18 @@ export class NeutralViewPali extends LitElement {
                 target="blank"
                 >SuttaCentral</a
               >
-              and are linked to the corresponding segments on that site in
-              table-view and numbers-view mode.
```

Comment:
Ha, this is not entirely correct any more for any of them. They are also linked in the text-view parallels ... that also counts for all other languages.